includes/definitions/discovery/firebrick.yaml

```
@@ -1,3 +1,29 @@
+mib: FIREBRICK-CPU-MIB:FIREBRICK-MONITORING
 modules:
+    processors:
+        data:
+            -
+                oid: fbCpuUsageTable.0
+                num_oid: '.1.3.6.1.4.1.24693.100.2.1.1.2.0.{{ $index }}'
+                type: firebrick-cpu
+                precision: 0.01
     os:
         sysDescr_regex: '/^(?<hardware>FB[0-9]{4}).*\((?<version>.*)\).*$/m'
```

Could change this to get rid of the timestamp in the version field
```suggestion
        sysDescr_regex: '/^(?<hardware>FB[0-9]{4}).*\((?<version>[^ ]+)/m'
```

---

That's a reasonable change - I'll make that in the next batch of changes I commit.

---

Hmmmm - that change has unintended side effects - I've just lost the version numbers off the test LibreNMS I'm running.....  More investigation needed as to why!

---

... Because I failed at writing the change....